src/webgl/p5.Camera.js

```diff
@@ -193,23 +193,81 @@ p5.prototype.perspective = function (...args) {
   return this;
 };
 
+
 /**
  *
  * Enable or disable perspective for lines in the WebGL renderer.
- * When linePerspective is enabled, lines will be affected by the current camera's perspective.
- * When linePerspective is disabled, lines will have a uniform scale regardless of the camera's perspective.
+ *
+ * The behavior of `linePerspective()`  is associated with the type of camera projection being used.
+ *
+ *  When using `perspective()`, which simulates realistic perspective, linePerspective
+ *  is set to `true` by default. This means that lines will be affected by the current
+ *  camera's perspective, resulting in a more natural appearance.
+ *
+ *  When using `ortho()` or `frustum()`, which do not simulate realistic perspective,
+ *  linePerspective is set to `false` by default. In this case, lines will have a uniform
+ *  scale regardless of the camera's perspective, providing a more predictable and
+ *  consistent appearance.
+ *
+ *  You can override the default behavior by explicitly calling `linePerspective()` after
+ *  using perspective(), ortho(), or frustum(). This allows you to customize the line
```

Comment:
Maybe we can do these as inline code too for consistency:
```suggestion
 *  using `perspective()`, `ortho()`, or `frustum()`. This allows you to customize the line
```